dcsazure_AzureSQL_to_AzureSQL_prof_pl/README.md

```diff
@@ -0,0 +1,80 @@
+# dcsazure_AzureSQL_to_AzureSQL_prof_pl
```

Comment:
Observed that in the [PR](#23)  for adding checkpointing and logging, the suffix for profiling pipelines has been changed from `_prof_pl` to `_discovery_pl`. 

@JessicaLHartog do we need to make similar change in this PR?

Reply:
Renamed the pipeline in #24.